### DISCUSSION TITLE
Bug on gist.github.com

### DISCUSSION THREAD

**Discussion by getify** [community/community#20287] [Other Feature Feedback, Questions, & Ideas] [answered]
<img width="597" alt="github-error" src="https://user-images.githubusercontent.com/150330/178150772-14bb5107-f868-4208-bf14-03eaaeaf92a0.png">

**Comment by radar-blip** [comment, marked as answer]
This has come up a few times. A small glitch. (clicking on notifications from gists gets 404 page?) 
Reports have been filed and hopefully it will be fixed soon.

If you would like to report it also, you can do so here: https://support.github.com/contact/bug-report

Have a great day!

**Reply by radar-blip** [reply]
Update: Seems to be fixed now.

**Comment by ws-jm** [comment]
ok. his say right.

**Comment by alimttth** [comment]
This has come up a few times. A small glitch. (clicking on notifications from gists gets 404 page?)
Reports have been filed and hopefully it will be fixed soon.

If you would like to report it also, you can do so here: https://support.github.com/contact/bug-report

Have a great day!
